File: JAVA/2011. Final Value of Variable After Performing Operations.java
class Solution {
    public int finalValueAfterOperations(String[] s) {
        int x=0,i;
        for(i=0;i<s.length;i++){
            if(s[i].equals("X++")||s[i].equals("++X"))
            x++;
            else
            x--;
        }
        return x;
    }
}

File: C++/2011. Final Value of Variable After Performing Operations.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& s) {
        int x=0;
        for(string c:s){
            if(c=="X++"||c=="++X")
            x++;
            else
            x--;
        }
        return x;
    }
};